dockerfiles/theia-dev/centos7.Dockerfile

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2018-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM centos:7
```

Comment:
hello, where is UBI here ?
AFAIK it's not useful to have centos theia-dev

Reply:
Unfortunately we cannot use UBI for builder image, because we lack some packages: `jq` and `libxkbfile-devel`

Reply:
I mean, it's a dev container so ubi is not helpful there. It's more useful into production container (eclipse/che-theia image)